lib/bolt/cli.rb

```diff
@@ -425,7 +426,7 @@ def execute(options)
         execute_plan(executor, options)
       else
         executor = Bolt::Executor.new(@config, options[:noop])
-        nodes = executor.from_uris(options[:nodes])
+        targets = options[:nodes]
```

Comment:
This part feels gross, but we use `--nodes` on the CLI so that name's going to be present somewhere. I'd really like to get ride of the `Node` class soon so it gets less confusing again.